crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -435,6 +435,14 @@ define_tedge_config! {
     },
 
     mqtt: {
+        /// MQTT topic root
+        #[tedge_config(default(value = "te"))]
+        topic_root: String,
+
+        /// The device MQTT topic identifier
+        #[tedge_config(default(value = "device/main//"))]
+        device_topic_id: String,
```

Comment:
@reubenmiller you told me off-line that you where not happy with this name.
 
 What would be your proposal?

Reply:
That's fine, we'll leave it as is